astroid/protocols.py

```diff
@@ -167,6 +167,9 @@ def _multiply_seq_by_int(
     context: InferenceContext,
 ) -> _TupleListNodeT:
     node = self.__class__(parent=opnode)
+    if other.value > 1e8:
```

Comment:
Let's not pass a `Const` but an `int`:
```console
>>> "" >1e8
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: '>' not supported between instances of 'str' and 'float'
>>>
```

There are two call sites, one is easy to fix. The other might need one more `isinstance`, but at least it won't crash and is type safe 😄

Reply:
I meant changing `other` to `int`. The function name indicates that it should always be `int`.